application/ui/components/params-list.jsx

```diff
@@ -68,16 +81,50 @@ module.exports = React.createClass({
         return this.renderParam(param, []);
     },
 
+    includeChanged : function(event)
+    {
+        var values, path, requestActions;
+
+        values = _.extend({}, this.props.requestValues);
+
+        path = event.currentTarget.dataset.path;
+
+        requestActions = this.getFlux().actions.request;
+
+        if (event.currentTarget.checked) {
+            requestActions.removeFromExcludedFields(this.props.methodName, path);
+        } else {
+            requestActions.addToExcludedFields(this.props.methodName, path);
+        }
+    },
+
     renderParam : function(param, path)
     {
-        var renderedMarkup;
+        var renderedMarkup, checkboxComponent;
 
         // Record the "path" of this param in the request body object
         path = this.appendPath(path, param.name);
 
+        if (path.length === 1) {
+            checkboxComponent = (
+                <td>
+                    <input
+                        type      = "checkbox"
+                        name      = {'include-' + param.name}
+                        data-path = {path[0]}
+                        onChange  = {this.includeChanged}
+                        checked   = {! _(this.state.excludedFields).contains(path[0])}
+                    />
+                </td>
+            );
+        } else {
+            checkboxComponent = null;
```

Comment:
I don't think so. If path.length is greater than 1 it means it's a sub-param of a hash or array field.